_posts/en/newsletters/2025-01-10-newsletter.md

```diff
@@ -0,0 +1,175 @@
+  could contribute to a solution.  Sanders mentions, and Anthony Towns
+  [expands][towns coinid] on, the `coinid` feature from the Chia
+  blockchain, showing how it could reduce the amount of data required
+  down to a constant amount.  Salvatore Ingala [posted][ingala cat]
+  about a similar mechanism using [OP_CAT][topic op_cat] that he learned
+  about from developer Rijndael, who later [provided details][rijndael
+  cat] himself.  Brandon Black [described][black penalty] an
+  alternative type of solution---a penalty-based variant of
+  LN-Symmetry---and cited work by Daniel Roberts about it (see next news
+  item).
+
+- **Multiparty LN-Symmetry variant with penalties for limiting published updates:**
+  Daniel Roberts [posted][roberts sympen] to Delving Bitcoin about
+  preventing a malicious channel counterparty (Mallory) from being able
+  to delay channel settlement by deliberately broadcasting old states at
+  a higher feerate than an honest counterparty (Bob) is paying for
+  confirmation of the final state.  In theory, Bob can rebind his final
+  state to Mallory's old state and both transactions might confirm in
+  the same block, causing her to lose money on fees and Bob to get the
+  final state confirmed for the same fee cost he was already willing to
+  pay.  However, if Mallory can repeatedly prevent Bob from learning
+  about her broadcasts of old states before they get confirmed, she can
+  prevent him from responding until an [HTLCs][topic htlc] in the
+  channel expire and Mallory is able to steal funds.
+
+  Roberts proposed a scheme that only allows a channel participant to
+  get a single state confirmed.  If a later state gets confirmed, the
+  participant who submitted the final state and any participant who
+  didn't submit any states can take the money of any participants who
+  submitted outdated states.
+
+  Unfortunately, after publishing the scheme, Roberts discovered and
+  self-disclosed a critical flaw in it: similar to the _2x delay
+  problem_ described in the previous news item, the last party to sign
+  is able to complete a state that no other party can complete, giving
+  the final signer exclusive access to the current final state.  If any
+  other party tries to close in the previous state, they will lose
+  money if the final signer uses the final state.
+
+  Roberts is investigating alternative approaches, but the topic spurred
+  interesting discussion about whether or not adding a penalty mechanism
+  to LN-Symmetry is useful.  Gregory Sanders, whose previous
+  proof-of-concept implementation of LN-Symmetry lead him to believe
+  that a penalty mechanism is unnecessary (see [Newsletter
+  #284][news284 sympen]), noted that the repeated-old-state attack is
+  similar to a [replacement cycling attack][topic replacement cycling].
+  He finds "this attack pretty weak, since the attack[er] can be driven
+  to negative EV [expected value] pretty easily" even if the defender
```

Comment:
```suggestion
  to negative EV [expected value] pretty easily," even if the defender
```
(the comma doesn't need to be part of the actual quote, even though it's within quotation marks)

Reply:
Leaving as-is.  I think it scans better without the comma.